lib/Form/Checkbox/Group.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Checkbox from './';
+
+export default class Group extends Component {
+  static propTypes = {
+    choices: PropTypes.arrayOf(PropTypes.shape({
+      label: PropTypes.string.isRequired,
+      subtitle: PropTypes.string,
+      id: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
+      checked: PropTypes.bool,
+    })).isRequired,
+    onChangeHandler: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    const selected = props.choices.filter(c => c.checked);
+    this.state = { choices: props.choices, selected };
+    this.onChangeHandler = this.onChangeHandler.bind(this);
+    this.removeChoice = this.removeChoice.bind(this);
+    this.addChoice = this.addChoice.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.choices !== this.props.choices) {
+      const selected = nextProps.choices.filter(c => c.checked);
+      this.setState({ choices: nextProps.choices, selected });
+    }
+  }
+
+  onChangeHandler(e) {
+    const choice = this.state.choices.filter(c => c.id === e.target.id).shift();
```

Comment:
Are we using `shift` to make sure we get one element only?

Reply:
yes, is there a better way to do this?

Reply:
I was just wondering if we should even worry about that? If it fails because it has more than 1 element I think we should see that error happen right?

What happens if this returns more than one choice and because we never allow that to error we get some corrupted experience that is very hard to debug?